lib/tasks/version.rake

```diff
@@ -0,0 +1,21 @@
+namespace :data do
+  desc 'Sets conference_id in all pre-existing PaperTrail::Version objects'
+  task set_conference_id: :environment do
```

Comment:
The name of this task is too general. This could refer to anything. 
The name of the task should tell us a bit more,eg, `set_conference_in_versions`, and it should be less generic. What if we want to set the conference_id attribute in other tables in the future... It will be confusing.